fusion-plugin-jwt/flow-typed/npm/node-fetch_v1.x.x.js

```diff
@@ -1,9 +1,11 @@
-// flow-typed signature: 284e255a331cbe00e3ddf88897c9452d
-// flow-typed version: 7e7beb7540/node-fetch_v1.x.x/flow_>=v0.44.x
+// @flow
```

Comment:
No libdef definition for `node-fetch@2.x.x` on flow-typed.  For now, this should do though (taken from `v1.x.x` with support for current version of Flow).